### DISCUSSION TITLE
Documentation: add info about default browser path

### DISCUSSION THREAD

**Discussion by tweinreich** [apache/superset#18542] [Ideas]
## Environment
- **OS:** Ubuntu 18.04.6 (Bionic Beaver)
- **Superset version:** 1.3.2

## Issue
When installing Firefox ESR on Ubuntu the binary is placed in `/usr/bin/firefox-esr`. If you try to send a report including a screenshot now, Superset will complain with this message:

> Error: Failed taking a screenshot 
> Message: Expected browser binary location, but unable to find binary in default location, no 'moz:firefoxOptions.binary' capability provided, and no binary flag set on the command line
> 

_* the error is sent per email and included in the reports execution log_

## Solution
To solve this, you can symlink this binary to the "default location": `/usr/local/bin/firefox`

In my case this solved the issue and I am receiving emails including screenshots now. It was difficult finding the "default location" but finally I found it [inside the Dockerfile](https://github.com/apache/superset/blob/ceecc7047bcbbaf6c73b4915431c3fb3964b020a/Dockerfile#L147).

## Ideas
- include the information about the "default location" of the browsers in the documentation
- automatically search for the binaries (e.g. using `which`) and warn the admin if none was found
- include an option in the reports admin GUI to provide the browser path manually

**Comment by sfirke** [comment]
Those first two ideas would have been helpful for me in debugging why my reports weren't working and seem like they would not be too difficult to implement 👍
